crates/fuel-core/src/schema/gas_price.rs

```diff
@@ -40,14 +38,20 @@ impl LatestGasPriceQuery {
         &self,
         ctx: &Context<'_>,
     ) -> async_graphql::Result<LatestGasPrice> {
-        let config = ctx.data_unchecked::<GraphQLConfig>();
-
+        let gas_price_provider = ctx.data_unchecked::<GasPriceProvider>();
         let query: &ReadView = ctx.data_unchecked();
+
         let latest_block: Block<_> = query.latest_block()?;
         let block_height = u32::from(*latest_block.header().height());
+        let gas_price = gas_price_provider
+            .known_gas_price(block_height.into())
```

Comment:
Is that gas price used by the block producer at the last block, or is that the gas price for the next block?

If it is gas price used for the last block, then maybe we can get it from the `Mint` transaction=)